.github/workflows/pr-comment-build-dispatcher.yaml

```
@@ -0,0 +1,70 @@
+name: "PR command build dispatcher"
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  pull-requests: write
+
+concurrency:
+  group: ${{ github.workflow }}-pr-command-${{ github.head_ref || github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  check_maintainer_membership:
+    # Only run if this is a PR comment that contains a valid command
+    if: |
+      ${{ github.event.issue.pull_request }} &&
+        (    contains(github.event.comment.body, '/update-sdk')
+          || contains(github.event.comment.body, '/build-image') )
+    name: Check if commenter is in the Flatcar maintainers team
+    outputs:
+      maintainers: steps.step1.output.maintainers
+    runs-on:
+      - ubuntu-latest
+    steps:
+      - name: Fetch members of the maintainers team
+        env:
+          requester: ${{ github.event.comment.user.login }}
+        shell: bash
+        run: |
+          curl -L --silent \
```

```suggestion
          set -euo pipefail
          curl --fail --show-error -L --silent \
```
When the API fails, we can exit with an error instead of having an empty maintainers file